src/wrapper/java_vm/vm.rs

```diff
@@ -349,7 +352,7 @@ impl JavaVM {
     pub fn get_env(&self) -> Result<JNIEnv> {
         let mut ptr = ptr::null_mut();
         unsafe {
-            let res = java_vm_unchecked!(self.0, GetEnv, &mut ptr, sys::JNI_VERSION_1_1);
+            let res = java_vm_call_unchecked!(self, v1_2, GetEnv, &mut ptr, sys::JNI_VERSION_1_2);
```

Comment:
I've created a separate issue to remind us to revisit this: https://github.com/jni-rs/jni-rs/issues/500

Reply:
Since I ended up making `get_env` changes to mark it unsafe and use `JNIEnv::from_raw_unchecked()` I took the opportunity to also have it accept a `version` argument.

The safety docs for `get_env` now require the caller to know that the `JavaVM` supports at least JNI 1.4.